.github/workflows/main.yml

```diff
@@ -11,18 +11,27 @@ jobs:
       run: rustup self update
     - name: Install Rust
       run: |
+        set -e
```

Comment:
This is not needed on GitHub Actions (it's already the default).

Reply:
ah, I had tested azure and travis, and I was assuming gha was the same as azure. thanks!  (trying to update everyone, and everyone uses different CI)

Reply:
ah, I had tested azure and travis, and I was assuming gha was the same as azure. thanks! (trying to update everyone, and everyone uses different CI)